modules/op-verifying-pipeline-run-pac.adoc

```diff
@@ -0,0 +1,25 @@
+// This module is included in the following assemblies:
+// * pac/managing-pipeline-runs-pac.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="verifying-pipeline-run-pac_{context}"]
+= Verifying a pipeline run
+
+You can verify that the {pac} resolver correctly processes the pipeline run definition that you created.
+
+.Prerequisites
+
+* You installed the `tkn` command line utility
+* You are logged into your {OCP} cluster using the `tkn` command line utility
```

Comment:
Technically you cannot login to the cluster using `tkn`, you need to use `oc` to login

Reply:
And after I log in with `oc` , `tkn` just works?

Reply:
yes